src/noit_check_tools.h

```diff
@@ -149,9 +149,12 @@ MTEV_HOOK_PROTO(check_postflight,
                 void *, closure,
                 (void *closure, noit_module_t *self, noit_check_t *check, noit_check_t *cause))
 
+static mtev_log_stream_t noit_cluster_debug = NULL;
```

Comment:
Put this in the if... it's not good practice to declare a static file variable in a header.  You want static function scope.